src/Views/Homepage.vala

```diff
@@ -71,7 +71,7 @@ public class AppCenter.Homepage : AbstractView {
             row_spacing = 12,
             homogeneous = true,
             max_children_per_line = 5,
-            min_children_per_line = 3
+            min_children_per_line = 1
```

Comment:
This is probably 1 by default so we probably don't need to specify this and can remove this line

Reply:
It sets to 0 by default; that seems to have the same behavior as setting to 1, but figured I'd highlight the difference in case it was relevant.